extensions/widgets/gradientrampeditor/gradientrampeditor.lcb

```diff
@@ -493,71 +494,94 @@ private handler paintRampEditors() returns nothing
       put xCoordFromGradientOffset((the offset of tStop) * 100) into tX
       put the bottom of mGradientRectangle into tY
 
+      if tStopIndex = mCurrentlySelectedStopIndex then
+         variable tSelStop as GradientStop
+         put true into tHasSelected
+         put  tStop into tSelStop
+         add 1 to tStopIndex
+         next repeat
+      end if
+
+      drawStop (tX, tY, tStop) -- BN
+
+      add 1 to tStopIndex
+
+   end repeat
+
+   if tHasSelected then
+      put xCoordFromGradientOffset((the offset of tSelStop) * 100) into tX
+      put the bottom of mGradientRectangle into tY
+
+      drawStop (tX, tY, tSelStop) -- BN
+
+      -- If this is the selected stop then draw an extra ramp point above the gradient and a line linking it.
+      put xCoordFromGradientOffset((the offset of tSelStop) * 100) into tX
+      put the bottom of mGradientRectangle into tY
+      --
+      -- if tStopIndex is mCurrentlySelectedStopIndex then
```

Comment:
@BerndN You can delete this line rather than just commenting it out. Also lines 518 and 519 should be deleted.